chef/cookbooks/ceph/recipes/conf.rb

```diff
@@ -45,6 +45,8 @@
 keystone_settings = {}
 if is_rgw && !(node[:ceph][:keystone_instance].nil? || node[:ceph][:keystone_instance].empty?)
   keystone_settings = KeystoneHelper.keystone_settings(node, @cookbook_name)
+  keystone_settings["s3_auth"] = true
+  keystone_settings["accepted_roles"] = ["rgw_admin", "Member", "_member_"]
```

Comment:
Hmm, this looks slightly awkward. (modifying the return value of a shared keystone settings helper). IMHO since this is only used in the config file template, I'd just set those values explicitely in the config template and not bother putting that into the keystone settings hash.